Cargo.toml

```diff
@@ -9,3 +9,51 @@ edition = "2024"
 
 [dependencies]
 proconio = "0.4.5"
+
+[[example]]
```

Comment:
are these needed? If cargo build/test enforces it;its okay. else I don't really wanna have to remember to add it here;